src/API.ts

```diff
@@ -22,11 +26,12 @@ class API extends ApplicationAPI {
       docs: [],
       v1: [
         CreateMatchingRouter,
-        CreateUserRouter,
         DeleteUserRouter,
         GetUserRouter,
         HelloRouter,
         UpdateUserRouter,
+        InitializeSessionRouter,
+        RefreshSessionRouter,
```

Comment:
Just to make sure, the `InitalizeSessionRouter` replaces the functionality of the `CreateUserRouter` correct?

Reply:
I added it back, but they do almost the same thing now. `CreateUserRouter` creates a user directly, the other one logs in with Google, creates a user if no user exists, and then generates a session. I think `InitializeSessionRouter` is the only router that would be used though.

Reply:
Ya I just wanted to confirm, I think we should keep it deleted since the create user router is probably never going to be used and will just cause confusion for anyone trying to understand the codebase.